spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/InitializationMethod.scala

```diff
@@ -129,6 +129,18 @@ object VariableFormat {
     }
   }
 
+  case object GP_KH_KW_IN_OUT extends VariableFormat {
```

Comment:
There is a one-line java doc explaining the `VariableFormat`. User are not supposed to use this unless he/she is implementing a layer or a IntializationMethod, right? I think the api is pretty self-explanatory. Do we need a page in the website?